client/src/frontend/containers/JournalDetail/index.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import PropTypes from "prop-types";
+import CheckFrontendMode from "global/containers/CheckFrontendMode";
+import EntityCollection from "frontend/components/composed/EntityCollection";
+import lh from "helpers/linkHandler";
+import HeadContent from "global/components/HeadContent";
+import EventTracker, { EVENTS } from "global/components/EventTracker";
+import {
+  useSelectSettings,
+  useSelectJournal,
+  usePaginationState,
+  useFilterState,
+  useUrlFromState
+} from "hooks";
+import { RegisterBreadcrumbs } from "global/components/atomic/Breadcrumbs";
+import Layout from "frontend/components/layout";
+import { pageChangeHandlerCreator } from "helpers/pageChangeHandlerCreator";
+
+export default function JournalDetailContainer({ location, match }) {
+  const { journal, journalResponse } = useSelectJournal(match);
+  const settings = useSelectSettings();
+  const { paginationState, handlePageChange } = usePaginationState(
+    location,
+    20
+  );
+  const { filterState, updateFilterState } = useFilterState(location, {
+    collectionOrder: match.params.id
+  });
+  // TODO: add useDispatchIssues hook
+  // TODO: add useSelectIssues hook
+  useUrlFromState(location, filterState, paginationState.number);
+  if (!journal) return null;
+
+  const ogTitle = () => {
+    if (!settings) return null;
+    const { socialTitle, title } = journal.attributes;
+    return (
+      socialTitle ||
+      `\u201c${title}\u201d on ${settings.attributes.general.installationName}`
+    );
+  };
+
+  const ogDescription = () => {
+    const { descriptionPlaintext, socialDescription } = journal.attributes;
+    return socialDescription || descriptionPlaintext;
+  };
+
+  const ogImage = () => {
+    const { socialImageStyles, heroStyles } = journal.attributes;
+    if (socialImageStyles?.mediumLandscape)
+      return socialImageStyles.mediumLandscape;
+    if (heroStyles?.mediumLandscape) return heroStyles.mediumLandscape;
+    return null;
+  };
+
+  return (
+    <div>
+      <CheckFrontendMode
+        debugLabel="ProjectCollectionDetail"
+        isProjectSubpage
+      />
```

Comment:
Check the `debugLabel` prop in the `CheckFrontendMode` component for these new routes and make sure they're accurate.